ignition/service/resourcedriver.py

```diff
@@ -198,6 +199,12 @@ def execute_lifecycle(self, **kwarg):
         try:
             logging_context.set_from_headers()
 
+            tenant_id=None
+            if('TenantId' in connexion.request.headers):
+                tenant_id = connexion.request.headers['TenantId']
+                logger.info("TenantId received in headers : %s", tenant_id)
```

Comment:
Not sure we want this to be logged at info level, as it'll be on every call. Maybe `debug` instead?

Reply:
@dvaccarosenna , I agree with you. I have changed the log level now.